Plugins/SimpleFileBrowser/README.txt

```diff
@@ -88,4 +88,4 @@ void FileBrowserHelpers.DeleteFile( string path );
 void FileBrowserHelpers.DeleteDirectory( string path );
 string FileBrowserHelpers.GetFilename( string path );
 long FileBrowserHelpers.GetFilesize( string path );
-DateTime FileBrowserHelpers.GetLastModifiedDate( string path );
\ No newline at end of file
+DateTime FileBrowserHelpers.GetLastModifiedDate( string path );
```

Comment:
Does this line use a different line ending? Why is it marked as changed?

Reply:
I did edit on Mac OS with github bundle editor.
I doesn't want to edit line of 91.